src/WaveClip.cpp

```diff
@@ -1670,7 +1669,7 @@ void WaveClip::SetRate(int rate)
 }
 
 /*! @excsafety{Strong} */
-void WaveClip::Resample(int rate, ProgressDialog *progress)
+void WaveClip::Resample(int rate, BasicUI::ProgressDialog *progress)
```

Comment:
Didn't notice such dependency earlier. Would be nice to remove UI dependencies from WaveClip, but it's not in the scope of this PR anyway

Reply:
Yes, I thought of just adding a `std::function<void()>` argument for progress notifications, so not even BasicUI is mentioned here.  That might look better.  Still this change is enough to make things platform neutral and keep library dependencies acyclic, and anyway there will be transitive dependency on lib-basic-ui because we use lib-exceptions.